nerium/query.py

```diff
@@ -91,7 +93,9 @@ def inner(query_name, *args, **kwargs):
             result = func(query_name, *args, **kwargs)
             query = query._replace(result=result)
         except Exception as e:
-            query = query._replace(error=repr(e), status_code=400)
+            error = traceback.format_exc() if DebugModeChecker.is_debug_mode() else repr(e)
+
+            query = query._replace(error=error, status_code=400)
```

Comment:
This allows the stack trace to be logged when the service is running in debug mode.